site/app/templates/grading/electronic/SolutionTaNotesPanel.twig

```diff
@@ -26,42 +26,32 @@
             <div class="component-title">
                 Solution for {{ component_title }} {% if is_itempool_linked %} (student having {{ itempool_item }}) {% endif %}
             </div>
-            <button type="primary"
-                    id="show-sol-btn-{{ component_id }}"
-                    class="btn btn-primary {% if not is_first_edit %} hide {% endif %}"
-                    onclick="showSolutionTextboxCont('#show-sol-btn-{{ component_id }}.btn', '#sol-textbox-cont-{{ component_id }}-edit', '.solution-notes-text-{{ component_id }}');"
-            >Start Now</button>
             <div class="last-edit {% if is_first_edit %} hide {% endif %}">
               Last edit @ <i class="last-edit-time">{{ last_edit }}</i> by <i class="last-edit-author">{{ author }}{% if current_user_id == author %} (You) {% endif %}</i>
             </div>
         </div>
         <div class="row divider"></div>
-        <div class="solution-cont">
+        <div class="solution-cont" data-component_id="{{component_id}}">
             <span class="solution-notes-text solution-notes-text-{{ component_id }} {% if not is_first_edit %} hide {% endif %}">
                 No Solutions or TA notes for  "{{ component_title}}" Component  {% if is_itempool_linked %} (student having {{ itempool_item }}) {% endif %}
             </span>
-            <div class="solution-saved {% if is_first_edit %} hide {% endif %}" id="sol-textbox-cont-{{ component_id }}-saved">
-                <span class="solution-notes-text">
-                    {{ solution_notes }}
-                </span>
-                <button type="primary"
-                        id="edit-solution-btn-{{ component_id }}"
-                        class="btn btn-primary"
-                        onclick="showSolutionTextboxCont('#edit-sol-btn-{{ component_id }}.btn', '#sol-textbox-cont-{{ component_id }}-edit', '#sol-textbox-cont-{{ component_id }}-saved');"
-                >Edit</button>
-            </div>
-            <div class="solution-edit hide" id="sol-textbox-cont-{{ component_id }}-edit">
+            <div class="solution-edit" id="sol-textbox-cont-{{ component_id }}-edit">
                 <label for="textbox-solution-{{ component_id }}" tabIndex="0" class="screen-reader">Solution for {{ component_title }}</label>
-                <textarea id="textbox-solution-{{ component_id }}"
-                          class="solution-ta-notes-textbox"
-                          placeholder='Start writing the solution for "{{ component_title }}" component.'
-                >{{ solution_notes }}</textarea>
+                {% set initialize_preview = not is_first_edit %}
```

Comment:
Is this needed? Can you directly do `"initialize_preview" : is_first_edit == true`